### PR TITLE
Bugfix-Multipage apps not working in Windows #52

### LINKED ISSUE #52: Multi-Page Apps [Bug] Doesn't work with Windows
As reported on the forum [here](https://community.plotly.com/t/introducing-dash-pages-a-dash-2-1-feature-preview/57775/35), the multi-page app doesn't work with Windows - the apps aren't in `dash.page_registry` dict

The problem is in [these 2 lines](https://github.com/plotly/dash-labs/blob/main/dash_labs/plugins/pages.py#L235-L236.) of code.  Using os.walk instead of glob for getting files in sub directories should fix it.  PR coming soon.

### PR DESCRIPTION
**Pull request by johnkangw**
Closes #52

Fixed so multipage apps work in windows

### CODE REVIEW

**Comment by AnnMarieW** [comment]
Hi @johnkangw Thanks for reporting this issue on the forum and helping with the solution.  This looks really good!

@chriddyp I can verify this now works with the three demo apps  on both Linux and Windows.  Adding tests remains an open issue.  Do you see anything else before this is merged?

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 dash >=2.0.0
+pandas >=1.3.4
```
Comment (alexcjohnson) [inline requirements.txt:2]: Intentional to add `pandas` here? In general we recommend but do not require it.

**Review by alexcjohnson — APPROVED** [review]
💃 This looks great @johnkangw - thanks! Just see my question about pandas.

**Comment by johnkangw** [comment]
Alex, thanks for the note. I intentionally added it in because when I ran the code without pandas pandas it gave me an error. 

> 💃 This looks great @johnkangw - thanks! Just see my question about pandas.

**Comment by AnnMarieW** [comment]
@alexcjohnson Pandas is only needed for some of the examples in the demo folder to work.  It's not required for the `pages.py` plug-in.   In that case should `pandas` be removed from  `requirements.txt`?

**Comment by alexcjohnson** [comment]
Thanks for the clarification @johnkangw @AnnMarieW - yes, if pandas is only used by the demos but not in the package itself I think we should remove it, but mention it in the installation section of the README like we do in https://dash.plotly.com/installation:

> We also recommend installing [Pandas](https://pandas.pydata.org/), which is required by [Plotly Express](https://plotly.com/python/plotly-express/) and used in many of our examples.

**Pull request merged**